apps/server/src/modules/user-login-migration/controller/dto/response/user-login-migration.response.ts

```diff
@@ -19,7 +19,7 @@ export class UserLoginMigrationResponse {
 	@ApiProperty({
 		description: 'Date when the migration was started',
 	})
-	startedAt: Date;
+	startedAt?: Date;
```

Comment:
why is startedAt optional?

Reply:
If we revert the userloginMigration we need to return it. We return a reverted userLoginMigration like this: 		const revertedUserLoginMigration = new UserLoginMigrationDO({
			schoolId: userLoginMigration.schoolId,
			targetSystemId: userLoginMigration.targetSystemId,
			startedAt: undefined,
			finishedAt: undefined,
			mandatorySince: undefined,
			closedAt: undefined,
			sourceSystemId: undefined,
		});

Reply:
when we revert it, it is deleted

Reply:
it is now required again.